web/libs/editor/src/regions/Result.js

```diff
@@ -60,9 +61,65 @@ const Result = types
       "pairwise",
       "videorectangle",
       "ranker",
+      ...Registry.customTags.map((t) => t.resultName),
+    ])) : types.enumeration([
+      "labels",
+      "hypertextlabels",
+      "paragraphlabels",
+      "rectangle",
+      "keypoint",
+      "polygon",
+      "brush",
+      "bitmask",
+      "ellipse",
+      "magicwand",
+      "rectanglelabels",
+      "keypointlabels",
+      "polygonlabels",
+      "brushlabels",
+      "bitmasklabels",
+      "ellipselabels",
+      "timeserieslabels",
+      "timelinelabels",
+      "choices",
+      "datetime",
+      "number",
+      "taxonomy",
+      "textarea",
+      "rating",
+      "pairwise",
+      "videorectangle",
+      "ranker",
+      ...Registry.customTags.map((t) => t.resultName),
     ]),
     // @todo much better to have just a value, not a hash with empty fields
-    value: types.model({
+    value: ff.isActive(ff.FF_CUSTOM_TAGS) ? types.late(() => types.model({
```

Comment:
The definitions of `model` for `value` are essentially the same.  They may only require a condition for adding custom tags, but in general we may avoid the duplicating.